### PR TITLE
Fix typos, mispelling, and converting double and single quotes to " and '

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: RELEASE_BLOG_POST_TEMPLATE.md.bak [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (christinaforney) [inline RELEASE_BLOG_POST_TEMPLATE.md.bak]: Did you bring this template back intentionally? Why are there two?

File: blogposts/5-easy-ways-to-start-contributing-to-docker-using-sourcegraph.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Reading the implementation of [DockerCli’s Command method](https://sourcegraph
 
 So, in the case of “docker build”, it calls the [CmdBuild](https://sourcegraph.com/github.com/docker/docker/-/def/GoPackage/github.com/docker/docker/api/client/-/DockerCli/CmdBuild) method, which we can navigate to using Sourcegraph.
 
-There’s a lot going on here. At the top of the method, we see code dealing with a variety of input methods for the Dockerfile and configuration. Oftentimes, a good strategy for reading through a long method is to work backwards. Start at the bottom and look at what the method does at the very end. In many cases, that’s the meat of the method and everything before is just setup for completing that core action.
+there's a lot going on here. At the top of the method, we see code dealing with a variety of input methods for the Dockerfile and configuration. Oftentimes, a good strategy for reading through a long method is to work backwards. Start at the bottom and look at what the method does at the very end. In many cases, that’s the meat of the method and everything before is just setup for completing that core action.
```
Comment (christinaforney) [inline blogposts/5-easy-ways-to-start-contributing-to-docker-using-sourcegraph.md]:
```suggestion
There's a lot going on here. At the top of the method, we see code dealing with a variety of input methods for the Dockerfile and configuration. Oftentimes, a good strategy for reading through a long method is to work backwards. Start at the bottom and look at what the method does at the very end. In many cases, that’s the meat of the method and everything before is just setup for completing that core action.
```

File: blogposts/5-short-stories-from-open-source-pains-in-gains.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Open source projects die or fade for a variety of reasons: maintainers move on,
 
 *   It might be depended on by something somewhere
 *   Security and trust are important values of open source and to ensure these, the code needs to be kept up to date.
-*   There’s potential to build new features and fix new bugs that can benefit others.
+*   there's potential to build new features and fix new bugs that can benefit others.
```
Comment (christinaforney) [inline blogposts/5-short-stories-from-open-source-pains-in-gains.md]:
```suggestion
*   There's potential to build new features and fix new bugs that can benefit others.
```

File: blogposts/announcing-sourcegraph-3.3.md [hunk 4]
```diff
@@ -189,7 +189,7 @@ See examples of how developers at companies such as Uber, Lyft, and Yelp depend
   <p style="text-align: center"><a href="https://vimeo.com/331346276" target="_blank">View on Vimeo</a></p>
 </p>
 
-[Sentry](https://sentry.io/) is an [open-source](https://github.com/getsentry/sentry) error tracking tool which captures and sends error and exception notifications instantly. When browsing code on Sourcegraph or your code host, wouldn’t it be great if you could jump straight to the Sentry records page whenever you see error handling code instead of manually navigating around Sentry?
+[Sentry](https://sentry.io/) is an [open-source](https://github.com/getsentry/sentry) error tracking tool which captures and sends error and exception notifications instantly. When browsing code on Sourcegraph or your code host, Wouldn't it be great if you could jump straight to the Sentry records page whenever you see error handling code instead of manually navigating around Sentry?
```
Comment (christinaforney) [inline blogposts/announcing-sourcegraph-3.3.md]:
```suggestion
[Sentry](https://sentry.io/) is an [open-source](https://github.com/getsentry/sentry) error tracking tool which captures and sends error and exception notifications instantly. When browsing code on Sourcegraph or your code host, wouldn't it be great if you could jump straight to the Sentry records page whenever you see error handling code instead of manually navigating around Sentry?
```

File: blogposts/building-a-product-one-user-interview-at-a-time.md [hunk 5]
```diff
@@ -86,7 +86,7 @@ The wrap up questions are used to cover any additional topics or follow-up on an
 
 ### Developing insights
 
-You’ve completed your research, started to see patterns in the questions, and you feel like a pro interviewer. But what now?
+you've completed your research, started to see patterns in the questions, and you feel like a pro interviewer. But what now?
```
Comment (christinaforney) [inline blogposts/building-a-product-one-user-interview-at-a-time.md]:
```suggestion
You've completed your research, started to see patterns in the questions, and you feel like a pro interviewer. But what now?
```

File: blogposts/github-universe-liveblog-ceos-keynote.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ Let’s talk about students. When Wanstrath was in college, he torrented a lot o
 
 _[Editor’s note: unfortunately, conference WiFi cut out at this part of the talk. We’re trying to get the slides to fill this in.]_
 
-There’s also GitHub Classroom. GitHub also sponsors Major League Hacking and other hackathons. They’re also partnering with Udacity and Black Girls Code.
+there's also GitHub Classroom. GitHub also sponsors Major League Hacking and other hackathons. They’re also partnering with Udacity and Black Girls Code.
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-ceos-keynote.md]:
```suggestion
There's also GitHub Classroom. GitHub also sponsors Major League Hacking and other hackathons. They’re also partnering with Udacity and Black Girls Code.
```

File: blogposts/github-universe-liveblog-ceos-keynote.md [hunk 7]
```diff
@@ -89,7 +89,7 @@ The White House is using GitHub. Introducing Alvand Salehi.
 
 GitHub has a lot of integrations and are investing in relationships with integration partners to grow the ecosystem. They have a new early access program for key partners to get access to beta, unreleased features.
 
-There’s a new GraphQL API, already released to selected integration partners.
+there's a new GraphQL API, already released to selected integration partners.
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-ceos-keynote.md]:
```suggestion
There's a new GraphQL API, already released to selected integration partners.
```

File: blogposts/github-universe-liveblog-innersource.md [hunk 8]
```diff
@@ -55,9 +55,9 @@ Q: What’s the single biggest challenge so far?
 
 Jeremy: They had a guy quit because he became the only expert on an area of the code. He spent all his time reviewing and responding to issues and couldn’t code anymore
 
-Yasuhiro: The firewall makes it more difficult to collaborate. If more things were open source, this wouldn’t be an issue.
+Yasuhiro: The firewall makes it more difficult to collaborate. If more things were open source, this Wouldn't be an issue.
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-innersource.md]:
```suggestion
Yasuhiro: The firewall makes it more difficult to collaborate. If more things were open source, this wouldn't be an issue.
```

File: blogposts/github-universe-liveblog-innersource.md [hunk 9]
```diff
@@ -55,9 +55,9 @@ Q: What’s the single biggest challenge so far?
...
 
-Joan: There’s a big culture shift required, especially for people accustomed to doing things a certain way.
+Joan: there's a big culture shift required, especially for people accustomed to doing things a certain way.
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-innersource.md]:
```suggestion
Joan: There's a big culture shift required, especially for people accustomed to doing things a certain way.
```

File: blogposts/github-universe-liveblog-innersource.md [hunk 10]
```diff
@@ -67,16 +67,16 @@ Joan: It would probably be security — if there was a security breach of so
 
 Q: Can you elaborate on how you’re dealing with security concerns?
 
-Jeff: We’re investing in automation to flag patterns that might indicate security breaches or code leakage. There’s still a long way to go in quantifying cost and risk (in some cases, it might be cheaper to just let someone steal code).
+Jeff: We’re investing in automation to flag patterns that might indicate security breaches or code leakage. there's still a long way to go in quantifying cost and risk (in some cases, it might be cheaper to just let someone steal code).
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-innersource.md]:
```suggestion
Jeff: We’re investing in automation to flag patterns that might indicate security breaches or code leakage. There's still a long way to go in quantifying cost and risk (in some cases, it might be cheaper to just let someone steal code).
```

File: blogposts/github-universe-liveblog-innersource.md [hunk 11]
```diff
@@ -67,16 +67,16 @@ Joan: It would probably be security — if there was a security breach of so
...
 
-Jeremy: There’s no going back. We just have to deal with it and learn more about encryption to make sure things are secure while they become more open.
+Jeremy: there's no going back. We just have to deal with it and learn more about encryption to make sure things are secure while they become more open.
```
Comment (christinaforney) [inline blogposts/github-universe-liveblog-innersource.md]:
```suggestion
Jeremy: There's no going back. We just have to deal with it and learn more about encryption to make sure things are secure while they become more open.
```

File: blogposts/gophercon-2018-an-over-engineering-disaster-with-macaroons.md [hunk 12]
```diff
@@ -130,7 +130,7 @@ that makes it much safer?
 Now, if only we could combine the strengths of both world, like having a Bearer token (like our car key) that
 could be attenuated or limited so it can be much safer.
 
-This would be like we handed the valet a key that wouldn’t work generally, but would work within a one-mile radius,
+This would be like we handed the valet a key that Wouldn't work generally, but would work within a one-mile radius,
```
Comment (christinaforney) [inline blogposts/gophercon-2018-an-over-engineering-disaster-with-macaroons.md]:
```suggestion
This would be like we handed the valet a key that wouldn't work generally, but would work within a one-mile radius,
```

File: blogposts/gophercon-2018-how-do-you-structure-your-go-apps.md [hunk 13]
```diff
@@ -46,7 +46,7 @@ Kat answers the opening question by building a beer reviewing service that has a
 - Ability to add some sample data.
 
 ## Types of Structure
-There’s a variety of ways one can dive in and start organizing a project. Some common ways of the beer reviewing service could be structured are:
+there's a variety of ways one can dive in and start organizing a project. Some common ways of the beer reviewing service could be structured are:
```
Comment (christinaforney) [inline blogposts/gophercon-2018-how-do-you-structure-your-go-apps.md]:
```suggestion
There's a variety of ways one can dive in and start organizing a project. Some common ways of the beer reviewing service could be structured are:
```

File: blogposts/gophercon-2018-how-do-you-structure-your-go-apps.md [hunk 14]
```diff
@@ -65,7 +65,7 @@ However, there are some issues with the group by function approach. For example,
     - Everything is grouped logically, but it’s still hard to decide which packages should be reused or when to make a new package.
 <img width="832" alt="screenshot 2018-08-28 16 02 01" src="https://user-images.githubusercontent.com/4897310/44753609-e6b54800-aadb-11e8-8d3e-73698cc0da4a.png">
 
-There’s a different approach that we can try - Grouping by context or also known as Domain Driven Development.
+there's a different approach that we can try - Grouping by context or also known as Domain Driven Development.
```
Comment (christinaforney) [inline blogposts/gophercon-2018-how-do-you-structure-your-go-apps.md]:
```suggestion
There's a different approach that we can try - Grouping by context or also known as Domain Driven Development.
```

File: blogposts/gophercon-2019-contributing-to-the-os-package-how-deep-do-you-go.md [hunk 15]
```diff
@@ -73,7 +73,7 @@ Long and meandering path to getting the solution merged. Problem, solution, deve
 * Time zone issue from being in Europe, so slower progress.
 * Go freezing deadline was coming up.
 * Go has extremely high standards.
-* Didn’t turn out as expected.
+* didn't turn out as expected.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-contributing-to-the-os-package-how-deep-do-you-go.md]:
```suggestion
* Didn't turn out as expected.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 16]
```diff
@@ -117,20 +117,20 @@ client:
 
 You’ll see a lot of these three-box examples in the talk. The first box shows the original code of the package. The second is the new code, the change. And the last is client code—code that’s outside the package, using it. In the client code, we’ll always call the package “pkg”.
 
-Don’t be confused by the word “client.” There’s no networking here; I just mean client in the more general sense of a user. 
+Don’t be confused by the word “client.” there's no networking here; I just mean client in the more general sense of a user.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
Don’t be confused by the word “client.” There's no networking here; I just mean client in the more general sense of a user. 
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 17]
```diff
@@ -117,20 +117,20 @@ client:
...
+Now the package author might be tempted to say, “You fool! I gave you a name for that struct! Why didn't you use it?” And I think that’s a perfectly reasonable response, though “fool” is a bit harsh. 
 
 
 Instead of saying that every possible bit of client code has to compile for a change to be compatible, we’ll exclude some problematic constructs. We just saw one: copying a struct literal or other type literal from inside the package. 
 
-Another one is the use of unkeyed struct literals, where you omit the field names when building a struct. Those are fine if you control the struct—if it’s in your own package, for instance. But don’t use them for structs outside your package. There’s even a vet check for that one.
+Another one is the use of unkeyed struct literals, where you omit the field names when building a struct. Those are fine if you control the struct—if it’s in your own package, for instance. But don’t use them for structs outside your package. there's even a vet check for that one.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
Another one is the use of unkeyed struct literals, where you omit the field names when building a struct. Those are fine if you control the struct—if it’s in your own package, for instance. But don’t use them for structs outside your package. There's even a vet check for that one.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 18]
```diff
@@ -240,7 +240,7 @@ Given two packages, an old and and a new, we go through their APIs and compare e
 
 We match symbols by name at this stage. And the fundamental rule of compatibility applies: if a name was removed, that’s an incompatible change; if a name was added, that’s a compatible change.
 
-It’s worth repeating: it’s always a compatible change to add a top-level symbol to your package. You can’t possibly break old code by doing that. There’s no way old code could even detect your new symbol, not even by reflection.
+It’s worth repeating: it’s always a compatible change to add a top-level symbol to your package. You can’t possibly break old code by doing that. there's no way old code could even detect your new symbol, not even by reflection.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
It’s worth repeating: it’s always a compatible change to add a top-level symbol to your package. You can’t possibly break old code by doing that. There's no way old code could even detect your new symbol, not even by reflection.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 19]
```diff
@@ -301,7 +301,7 @@ Two are _value_ methods: the receiver is the type itself, Number.
 Two are _pointer_ methods: the receiver is a pointer to Number.
 
 We’ll call all the exported methods of a type its exported method set. 
-There’s the exported method set of Number, and the exported method set of pointer to Number.
+there's the exported method set of Number, and the exported method set of pointer to Number.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
There's the exported method set of Number, and the exported method set of pointer to Number.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 20]
```diff
@@ -325,7 +325,7 @@ But you can change a pointer method to a value method. That works because the po
 
 How does the client code work with the new version of the package, where P is a value method? Go dereferences the pointer to get a value, then calls the method on that value.
 
-There’s a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
+there's a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
There's a deliberate asymmetry in the language here. You can always call a value method on a pointer, because you can always dereference a pointer to get a value.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 21]
```diff
@@ -336,9 +336,9 @@ We’ve talked about defined types. Let’s move on to channels.
 
 **Channels**
 
-There’s only one compatible change you can make to a channel type: you can remove the direction marker. 
+there's only one compatible change you can make to a channel type: you can remove the direction marker.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
There's only one compatible change you can make to a channel type: you can remove the direction marker. 
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 22]
```diff
@@ -336,9 +336,9 @@ We’ve talked about defined types. Let’s move on to channels.
 
 **Channels**
 
-There’s only one compatible change you can make to a channel type: you can remove the direction marker. 
+there's only one compatible change you can make to a channel type: you can remove the direction marker. 
 
-For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel wouldn’t have compiled in the first place.
+For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel Wouldn't have compiled in the first place.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
For example, if you started with a channel that you can only send on, you can change that to a channel that can either send or receive. That’s clearly safe, because any code that tried to receive from the channel wouldn't have compiled in the first place.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 23]
```diff
@@ -441,9 +441,9 @@ One last category of types to talk about: numeric types. You can compatibly “w
 
 Now wait a minute, you say. That can’t be right. I can’t change an int32 to an int64 in my package without potentially breaking clients, as this example shows.
 
-That’s true. There’s something I haven’t mentioned yet. It turns out to simplify things immensely. The compatibility rule for numeric types — in fact, _all the **type** compatibility rules I’ve been telling you about_ — assume that the type in question is the underlying type of a defined type. Or in plain English, that the type has a name.
+That’s true. there's something I haven’t mentioned yet. It turns out to simplify things immensely. The compatibility rule for numeric types — in fact, _all the **type** compatibility rules I've been telling you about_ — assume that the type in question is the underlying type of a defined type. Or in plain English, that the type has a name.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
That’s true. There's something I haven’t mentioned yet. It turns out to simplify things immensely. The compatibility rule for numeric types — in fact, _all the **type** compatibility rules I've been telling you about_ — assume that the type in question is the underlying type of a defined type. Or in plain English, that the type has a name.
```

File: blogposts/gophercon-2019-detecting-incompatible-api-changes.md [hunk 24]
```diff
@@ -487,7 +487,7 @@ Happily, the package compatibility rules we just covered do most of the work. We
 
 We only care about packages you can see outside the module, so internal packages don’t matter.
 
-There’s one wrinkle, an interesting violation of the Fundamental Rule of Compatibility: you can remove a package from a module and maintain compatibility, as long as you move it to another module and make sure that your original module requires that other module. In order for the package import path to stay the same, that other module must be nested inside the first. 
+there's one wrinkle, an interesting violation of the Fundamental Rule of Compatibility: you can remove a package from a module and maintain compatibility, as long as you move it to another module and make sure that your original module requires that other module. In order for the package import path to stay the same, that other module must be nested inside the first.
```
Comment (christinaforney) [inline blogposts/gophercon-2019-detecting-incompatible-api-changes.md]:
```suggestion
There's one wrinkle, an interesting violation of the Fundamental Rule of Compatibility: you can remove a package from a module and maintain compatibility, as long as you move it to another module and make sure that your original module requires that other module. In order for the package import path to stay the same, that other module must be nested inside the first. 
```

File: blogposts/how-to-make-your-open-source-project-thrive-with-andrey-petrov.md [hunk 25]
```diff
@@ -21,9 +21,9 @@ There are many benefits to open-sourcing your code. Here are just a few:
 *   It helps give you a bigger footprint of influence in the programming world.
 *   It lets you get the most out of sunk costs.
 
-You’ve invested a lot of time and effort into your code. Open-sourcing it makes it available and reusable. Employees come and go, as do companies, but open-source code will always be available to you and others who find it useful.
+you've invested a lot of time and effort into your code. Open-sourcing it makes it available and reusable. Employees come and go, as do companies, but open-source code will always be available to you and others who find it useful.
```
Comment (christinaforney) [inline blogposts/how-to-make-your-open-source-project-thrive-with-andrey-petrov.md]:
```suggestion
You've invested a lot of time and effort into your code. Open-sourcing it makes it available and reusable. Employees come and go, as do companies, but open-source code will always be available to you and others who find it useful.
```

File: blogposts/improving-language-support-in-2019.md [hunk 26]
```diff
@@ -15,13 +15,13 @@ Sourcegraph's [master plan](https://about.sourcegraph.com/plan) is to bring basi
...
+- Aside from Swift, we simply didn't prioritize work on language servers (partly because of the complexity of lsp-proxy, partly because of other priorities)
 
 Many things have changed in the last year to make it possible to improve language support now:
 
 - With [Sourcegraph extensions](https://docs.sourcegraph.com/extensions), it’s easier to understand how code intelligence on Sourcegraph works, which makes it easier to build on and all you need to understand is the Sourcegraph extension API (no need to understand [xfiles/xcontents](https://github.com/sourcegraph/language-server-protocol/blob/master/extension-files.md), [xcache](https://github.com/sourcegraph/language-server-protocol/blob/master/extension-cache.md), [lsp-adapter](https://github.com/sourcegraph/lsp-adapter), lsp-proxy, etc.) in order to add language support
-- There’s a new [Swift language server (apple/sourcekit-lsp)](https://github.com/apple/sourcekit-lsp)
-- There’s a new [Python language server (Microsoft/python-language-server)](https://github.com/Microsoft/python-language-server)
+- there's a new [Swift language server (apple/sourcekit-lsp)](https://github.com/apple/sourcekit-lsp)
```
Comment (christinaforney) [inline blogposts/improving-language-support-in-2019.md]:
```suggestion
- There's a new [Swift language server (apple/sourcekit-lsp)](https://github.com/apple/sourcekit-lsp)
```

File: blogposts/improving-language-support-in-2019.md [hunk 27]
```diff
@@ -15,13 +15,13 @@ Sourcegraph's [master plan](https://about.sourcegraph.com/plan) is to bring basi
...
+- there's a new [Python language server (Microsoft/python-language-server)](https://github.com/Microsoft/python-language-server)
```
Comment (christinaforney) [inline blogposts/improving-language-support-in-2019.md]:
```suggestion
- There's a new [Python language server (Microsoft/python-language-server)](https://github.com/Microsoft/python-language-server)
```

File: blogposts/part-1-how-sourcegraph-scales-with-the-language-server-protocol.md [hunk 28]
```diff
@@ -67,7 +67,7 @@ But that’s not the world we live in. Why not? Because to make every editor sup
 
 M x N
 
-LSP defines a communication protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. You’ve reduced the M x N problem to an M + N problem.
+LSP defines a communication protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. you've reduced the M x N problem to an M + N problem.
```
Comment (christinaforney) [inline blogposts/part-1-how-sourcegraph-scales-with-the-language-server-protocol.md]:
```suggestion
LSP defines a communication protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. You've reduced the M x N problem to an M + N problem.
```

File: blogposts/poetically-simple-code-review.md [hunk 29]
```diff
@@ -20,7 +20,7 @@ Hmm. LGTM?
 
 ![Code review is tedious.](//images.contentful.com/le3mxztn6yoo/3fbdV3zjyMCAI4s0iegQaK/14672e9928b60bcd0c06650713e694db/1_KbPOkjIgK5Y9-znSHL0ymg.png)
 
-There’s a better way
+there's a better way
```
Comment (christinaforney) [inline blogposts/poetically-simple-code-review.md]:
```suggestion
There's a better way
```

File: blogposts/poetically-simple-code-review.md [hunk 30]
```diff
@@ -52,7 +52,7 @@ Stop context-switching
 and getting lost line by line
 in so many tabs.
 
-There’s a better way 
+there's a better way
```
Comment (christinaforney) [inline blogposts/poetically-simple-code-review.md]:
```suggestion
There's a better way 
```

File: blogposts/sourcegraph-code-intelligence-and-the-language-server-protocol.md [hunk 31]
```diff
@@ -50,7 +50,7 @@ Historically, the problem of supporting multiple languages in multiple editors h
 
 Defining a clear protocol between the language analyzer and the code viewer/editor is crucial. Separating these concerns means no language-specific code at the application layer. The folks working on the frontend can focus on user experience without worrying about language semantics. The folks working on language analysis have a clear interface and don’t have to worry about the mechanics of the frontend.
 
-LSP defines such a protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. You’ve reduced a O(M·N) problem to a O(M+N) problem.
+LSP defines such a protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. you've reduced a O(M·N) problem to a O(M+N) problem.
```
Comment (christinaforney) [inline blogposts/sourcegraph-code-intelligence-and-the-language-server-protocol.md]:
```suggestion
LSP defines such a protocol that sits between editor plugins and the underlying analysis libraries. You build one language server for each language and one plugin for each editor. Each editor plugin that speaks LSP will now have support for every single language server. You've reduced a O(M·N) problem to a O(M+N) problem.
```

File: blogposts/sourcegraph-the-best-way-to-read-code-just-got-better.md [hunk 32]
```diff
@@ -50,7 +50,7 @@ If you want to see how something is used, you can quickly find and flip through
 
 [![](https://cdn-images-1.medium.com/max/1000/1*e31UMNeXH2iZjneS2bbUkg.png)](https://sourcegraph.com/github.com/golang/go/-/blob/src/net/http/request.go#L690)Find refs.
 
-There’s a new quick-launch bar that lets you jump to any symbol or file faster than you can say “global graph of code.” Activate it by clicking “Search” in the upper-right corner or just by typing “/” on any page.
+there's a new quick-launch bar that lets you jump to any symbol or file faster than you can say “global graph of code.” Activate it by clicking “Search” in the upper-right corner or just by typing “/” on any page.
```
Comment (christinaforney) [inline blogposts/sourcegraph-the-best-way-to-read-code-just-got-better.md]:
```suggestion
There's a new quick-launch bar that lets you jump to any symbol or file faster than you can say “global graph of code.” Activate it by clicking “Search” in the upper-right corner or just by typing “/” on any page.
```

File: blogposts/thyme-a-simple-cli-to-measure-human-time-and-focus.md [hunk 33]
```diff
@@ -67,7 +67,7 @@ Observations:
...
-*   There’s a solid 45-minute block of blue in the “Active” row between 16:30 and 17:30\. I remember this period. It was when I wrote 70% of the code and it felt amazing. I was mainly referencing [Sourcegraph](https://sourcegraph.com/) ([“template.Must — golang/go”](https://sourcegraph.com/github.com/golang/go/-/info/GoPackage/text/template/-/Must)) and the Google charts documentation. Everything I needed was in the Chrome window next to my editor so I was in the zone.
+*   there's a solid 45-minute block of blue in the “Active” row between 16:30 and 17:30\. I remember this period. It was when I wrote 70% of the code and it felt amazing. I was mainly referencing [Sourcegraph](https://sourcegraph.com/) ([“template.Must — golang/go”](https://sourcegraph.com/github.com/golang/go/-/info/GoPackage/text/template/-/Must)) and the Google charts documentation. Everything I needed was in the Chrome window next to my editor so I was in the zone.
```
Comment (christinaforney) [inline blogposts/thyme-a-simple-cli-to-measure-human-time-and-focus.md]:
```suggestion
*   There's a solid 45-minute block of blue in the “Active” row between 16:30 and 17:30\. I remember this period. It was when I wrote 70% of the code and it felt amazing. I was mainly referencing [Sourcegraph](https://sourcegraph.com/) ([“template.Must — golang/go”](https://sourcegraph.com/github.com/golang/go/-/info/GoPackage/text/template/-/Must)) and the Google charts documentation. Everything I needed was in the Chrome window next to my editor so I was in the zone.
```

File: blogposts/toward-a-url-for-every-function-in-the-world.md [hunk 34]
```diff
@@ -12,7 +12,7 @@ published: true
 
 
 
-What’s the best semantic and future-proof way to link to a piece of code? There’s been some interesting [discussion](https://news.ycombinator.com/item?id=8046710) about [the right way](http://andrew.yurisich.com/work/2014/07/16/dont-link-that-line-number/), and it’s something we think about a lot at [Sourcegraph](https://sourcegraph.com) as we build a better way for developers to discover and understand code.
+What’s the best semantic and future-proof way to link to a piece of code? there's been some interesting [discussion](https://news.ycombinator.com/item?id=8046710) about [the right way](http://andrew.yurisich.com/work/2014/07/16/dont-link-that-line-number/), and it’s something we think about a lot at [Sourcegraph](https://sourcegraph.com) as we build a better way for developers to discover and understand code.
```
Comment (christinaforney) [inline blogposts/toward-a-url-for-every-function-in-the-world.md]:
```suggestion
What’s the best semantic and future-proof way to link to a piece of code? There's been some interesting [discussion](https://news.ycombinator.com/item?id=8046710) about [the right way](http://andrew.yurisich.com/work/2014/07/16/dont-link-that-line-number/), and it’s something we think about a lot at [Sourcegraph](https://sourcegraph.com) as we build a better way for developers to discover and understand code.
```

File: RELEASE_BLOG_POST_TEMPLATE.md.bak (hunk 1 above)
Comment (ryan-blunden) [reply RELEASE_BLOG_POST_TEMPLATE.md.bak]: > Did you bring this template back intentionally? Why are there two?

No, I've deleted it now.
